translations/en-us.yml

```diff
@@ -1943,6 +1943,7 @@ osf-components:
         number_selected: '{numberOfFilesSelected} item(s) selected'
         clear_selection: 'Clear selection'
         checked_out_file: 'This file is checked out. Functionality is limited until it is checked in.'
+        download_count: '{count, plural, =0 {# Download} one {# Download} other {# Downloads}}'
```

Comment:
I think it should be `0 downloads` rather than `0 download`.

Reply:
English is a joke. :rofl:

Reply:
I know, right?